flow360/component/flow360_params/flow360_params.py

```diff
@@ -479,7 +479,7 @@ class Geometry(Flow360BaseModel):
     mesh_unit: Optional[LengthType.Positive] = pd.Field(alias="meshUnit")
 
     if Flags.beta_features():
-        decomposed_mesh: Optional[bool] = pd.Field(alias="decomposedMesh", default=False)
+        decomposed_mesh: Optional[bool] = pd.Field(alias="decomposedMesh")
```

Comment:
What is the reason that the default is moved out of the field definition?

Reply:
Just to be consistent with mesh_unit and the rest.

Reply:
Ok the GeometryLegacy one has the default in the field definition.